src/draw.ts

```diff
@@ -138,7 +138,7 @@ function addShape(drawable: Drawable, cur: DrawCurrent): void {
     drawable.shapes.push({
       orig: cur.orig,
       dest: cur.dest,
-      brush: cur.brush,
+      brush: cur.brush as cg.BrushColor,
```

Comment:
Let's use the new type for also for `cur.brush`, instead.

Reply:
This would require also changing the DrawCurrent type. Which causes problems with this line:
https://github.com/lichess-org/chessground/blob/755388ed3a84593eb99396ba616f47c41fa438d0/src/draw.ts#L58

new:  
```js
const brushes: cg.BrushColor[] = ['red', 'green', 'blue', 'paleBlue', 'yellow', 'paleGreen', 'paleRed', 'paleGrey'];
```

and also this:
https://github.com/lichess-org/chessground/blob/755388ed3a84593eb99396ba616f47c41fa438d0/src/draw.ts#L127-L131

new:
```js
function eventBrush(e: cg.MouchEvent): cg.BrushColor {
  const modA = (e.shiftKey || e.ctrlKey) && isRightButton(e);
  const modB = e.altKey || e.metaKey || e.getModifierState?.('AltGraph');
  return brushes[(modA ? 1 : 0) + (modB ? 2 : 0)];
}
```
Does this cause any problems? I dont know what the function eventBrush exactly does.

I could also create a Partial type for the brushes array, if it should stay unchanged.

```js
const brushes: Partial<cg.BrushColor>[] = ['green', 'red', 'blue', 'yellow'];
```
then only the return type of eventBrush needs to be changed.

```js
function eventBrush(e: cg.MouchEvent): cg.BrushColor {
  const modA = (e.shiftKey || e.ctrlKey) && isRightButton(e);
  const modB = e.altKey || e.metaKey || e.getModifierState?.('AltGraph');
  return brushes[(modA ? 1 : 0) + (modB ? 2 : 0)];
}
```

What do you prefer?

Reply:
i added a partial type, this shouldn't cause any conflicts